spec/features/versions_spec.rb

```diff
@@ -218,12 +218,21 @@
   scenario 'display changes in splashpages', feature: true, versioning: true, js: true do
     visit admin_conference_splashpage_path(conference.short_title)
     click_link 'Create Splashpage'
+    uncheck('Display tracks')
```

Comment:
instead of unchecking everything when creating it, what about leaving by default here and after clicking the edit link unchecking everything except _Make splash page public_, _Display tracks on the splashpage?_ and _Display the registration period on the splashpage?_